holopy/core/holopy_object.py

```diff
@@ -61,10 +62,14 @@ class HoloPyObject(Serializable):
 
     @property
     def _dict(self):
-        return dict(self._iteritems())
+        dictlist = dict(self._iteritems())
+        if self.__class__.__name__ is not 'Optics':
+            dictlist = dict_without(dictlist, ('wavelen', 'index', 'polarization'))
+        return dictlist
```

Comment:
I don't think we want to be doing something this invasive way up at the root HoloPyObject. If you need to do something like this, override the _dict method at the point where this becomes necessary (probably this is for marray or schema?)